sklearn/semi_supervised/_label_propagation.py

```diff
@@ -289,6 +289,8 @@ def fit(self, X, y):
             )
             self.n_iter_ += 1
 
+        l_bool_zeros = np.sum(self.label_distributions_, axis=1) == 0
+        self.label_distributions_[l_bool_zeros, :] = 1
```

Comment:
Why are you modifying `label_distributions_` and not just `normalizer`?

Reply:
because `normalizer ` is computed from `self.label_distributions_`, when the latter is all zeros, also the former is zero and then it tries to performs 0/0. Maybe it is better to modify only `normalizer` to set it to one when  `self.label_distributions_` is all zeros?

Reply:
Yes, that is what I would suggest.